plugins/vpp/l3plugin/vppcalls/vpp2001/dhcpproxy_dump.go

```diff
@@ -24,7 +24,7 @@ import (
 
 func (h *DHCPProxyHandler) DumpDHCPProxy() ([]*vppcalls.DHCPProxyDetails, error) {
 	var entry []*vppcalls.DHCPProxyDetails
-	reqCtx := h.callsChannel.SendMultiRequest(&vpp_dhcp.DHCPProxyDump{IsIP6: 0})
+	reqCtx := h.callsChannel.SendMultiRequest(&vpp_dhcp.DHCPProxyDump{})
```

Comment:
I think that explicitely setting `IsIP6` to `0` (`false`) was indicating that there will be another request for IPv6. So this could at least be commented to explain that we dump IPv4 first and IPv6 later.